### ISSUE TITLE
onConflict uses fully qualified column names in target conflict "where" clause, should be unqualified

### ISSUE THREAD

**Issue by laurence-myers** [laurence-myers/rose#8]
The column names should not be prefixed with the table name, as it's invalid syntax.

BAD:
```typescript
INSERT INTO
    "foo" as "t1"
("bar")
ON CONFLICT ("baz")
WHERE "t1"."qux" IS NULL DO NOTHING;
```

GOOD:
```typescript
INSERT INTO
    "foo" as "t1"
("bar")
ON CONFLICT ("baz")
WHERE "qux" IS NULL DO NOTHING;
```

**Comment by laurence-myers** [comment]
Fixed in 38c1f3a29be2d74b250238b25dcc1144f6a38637

**Comment by laurence-myers** [comment]
Fixed in v0.0.18
